src/plugins/status_page/public/status_page.html

```diff
@@ -15,27 +15,29 @@
     </div>
   </div>
 
-  <div class="row plugin_status_wrapper">
-    <h3>Installed Plugins</h3>
-    <div ng-if="!ui.statuses && ui.loading" class="loading_statuses">
+  <div class="row statuses_wrapper">
+    <h3>Status Breakdown</h3>
+
+    <div ng-if="!ui.statuses && ui.loading" class="statuses_loading">
       <span class="spinner"></span>
     </div>
 
-    <h4 ng-if="!ui.statuses && !ui.loading" class="missing_statuses">
-      No plugin status information available
+    <h4 ng-if="!ui.statuses && !ui.loading" class="statuses_missing">
+      No status information available
     </h4>
 
-    <table class="plugin_status_breakdown" ng-if="ui.statuses">
+    <table class="statuses" data-test-subj="statusBreakdown" ng-if="ui.statuses">
```

Comment:
Never mind, found the answer at https://github.com/elastic/kibana/pull/7459/files#diff-f8a58592f3622c5e7bf777330afcf177R17 :)